### ISSUE TITLE
Console doesn't show stream data for large streams

### ISSUE THREAD

**Issue by nitisht** [parseablehq/console#52]
To see this in action, open the Parseable demo site:

https://demo.parseable.io (creds: admin,admin). Select the `minioaudit` stream and select the time range to two months. You should see an empty page.

But if you query the same stream via curl (with command below), you'll see data present there.

```
curl --location 'https://demo.parseable.io/api/v1/query' \
--header 'Authorization: Basic YWRtaW46YWRtaW4=' \
--header 'Content-Type: application/json' \
--data '{
    "query":"select COUNT(*) from minioaudit",
    "startTime":"2023-02-14T00:00:00+00:00",
    "endTime":"2023-02-20T23:59:00+00:00"
}'
```

**Comment by Ryiski** [comment]
Its probably frozen trying to render a big batch of list

Plus based on the network tab, 1000 items per request is a no-no

![Screenshot from 2023-05-23 23-12-03](https://github.com/parseablehq/console/assets/47484379/06b0f49a-f74f-4e75-a594-1d63fe6436de)

And it's always running query twice the first time 

![Screenshot from 2023-05-23 23-17-24](https://github.com/parseablehq/console/assets/47484379/65b8a12c-3a63-4357-8630-0cad97535474)

**Comment by nitisht** [comment]
This is now fixed with #130 also related issue #137
